code/src/WijmoProvider/Features/GroupPanel.ts

```diff
@@ -70,6 +70,34 @@ namespace WijmoProvider.Feature {
                   this._dragCol && this._addGroup(this._dragCol, e);
         }
 
+        public addColumnsToGroupPanel(bindingList: string): void {
+            const groupDescriptions =
+                this._grid.provider.collectionView.groupDescriptions; // Group array
+            const columnList = JSON.parse(bindingList);
+            const source = this._grid.provider.itemsSource;
+            source.deferUpdate(() => {
+                for (let index = 0; index < columnList.length; index++) {
```

Comment:
You have a code-smell here: https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=241&id=OutSystems_outsystems-datagrid-reactive&open=AYDYHuR6Ao-lrqPVg1i3

Reply:
I'll change this as suggested.